networking/net_plugin.go

```diff
@@ -116,6 +119,7 @@ func (e *podEnv) execNetPlugin(cmd string, n *activeNet, netns string) ([]byte,
 	}
 
 	vars := [][2]string{
+		{"CNI_VERSION", "0.3.0"},
```

Comment:
We are already compliant with version 0.3.0, except for one thing: we omitted the version number. So, I'm fixing that.